assets/js/src/core/components/tag/tag.styles.ts

```diff
@@ -0,0 +1,35 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import { createStyles } from 'antd-style'
+
+export const useStyles = createStyles(({ css, token }) => {
+  return {
+    tag: css`
+        .anticon + span {
+            margin-inline-start: 4px;
+        }
+
+        &.ant-tag.theme-transparent {
+            background-color: transparent;
+        }
+
+        &.ant-tag.theme-user-role,
+        &.ant-tag.theme-admin-role {
+            background-color: rgba(0, 0, 0, 0.04);
+            color: #565656;
+            border-color: #EBEBEB;
+        }
```

Comment:
Do they really differ from the default color variant?

Reply:
@vin0401 This depends if you want to take the story book as the be all and end all. 
The colors are definitely different. 
I spoke about this with ida last week and have just followed up to get an idea how accurate to be going forward.

Reply:
confirmed that I can use the specific styling as the default as well.
Change incoming